alias-config.txt

```diff
@@ -0,0 +1,6 @@
+build: docker compose build
+start:docker compose up
+stop:docker compose down
+build-linter:docker compose build linter
+lint:docker compose run linter
+test:docker compose run webpack npm run test
```

Comment:
This is usable but not very extensible. For example, if I want to do `docker compose build --no-cache`, I would need to create a new command, maybe build-no-cache, which starts to defeat the purpose of the script.

I think it is a good improvement to add kwargs or args support so we can run build commands with args and flags. Thoughts?

```
sh alias.sh build
sh alias.sh build --no-cache
sh alias.sh build linter
sh alias.sh build linter --no-cache
```

Reply:
I think allowing for additional args and flags also defeats the purpose of the script assuming that the intention is to offer devs a more efficient method for executing longer commands without having to remember all of the necessary arguments and/or flags

I'm curious if @bzzz-coding has a preference or alternative perspective to offer?